core/embed/bootloader/CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## 2.1.4 [November 2023]
+
+### Added
+- Minimize risk of losing seed when upgrading firmware  [#2794]
+- Support interaction-less upgrade  [#2919]
```

Comment:
nit
```suggestion
- Minimize risk of losing seed when upgrading firmware.  [#2794]
- Support interaction-less upgrade.  [#2919]
```

Reply:
fixed in rebase. we should specify this as policy and ideally auto-check or auto-add the periods